PUBLISHING.md

```diff
@@ -0,0 +1,393 @@
+(The same region list will be used to determine where to publish SSM parameters.)
+
+Here's an example configuration for regions and the SSM prefix:
+
+```toml
+[aws]
+regions = ["us-west-2", "us-east-1", "us-east-2"]
+ssm_prefix = "/your/prefix/here"
+```
+
+This prefix forms the start of the name of each SSM parameter we set.
+The rest of the name comes from parameter templates.
+
+Parameter templates determine the name and value of each parameter we want to set for each AMI we've built.
+The [default template](tools/pubsys/policies/ssm/defaults.toml) creates parameters that let users find the AMI ID and the image version for each of your AMIs.
+The templates have access to the name of the current variant, architecture, etc., so they can create unique parameter names for each build.
+For more information on how templates work, check out [their documentation](tools/pubsys/policies/ssm/).
+
+If you're happy with the default template, you can set SSM parameters like this:
+
+```shell
+cargo make ssm
+```
+
+This will create versioned parameters, meaning that the parameter name has the image version in it.
+This isn't very discoverable yet, but it's useful for testing.
+
+As an example, a parameter might look like this:
+
+```
+/your/prefix/here/aws-k8s-1.17/x86_64/1.0.1-dafe3b16/image_id
+```
+
+Once you're satisfied with your image and parameters, you can promote the parameters to simpler names (for example, "latest") using the [instructions below](#promoting-ssm-parameters).
+
+Note: if you want to customize the SSM parameters that get set, you can copy and modify the existing template file, then point to your file like this:
+
+```shell
+cargo make ssm -e PUBLISH_SSM_TEMPLATES_PATH=/my/template/path
+```
+
+## Making your AMIs public
+
+We talked about [granting AMI access](#granting-access-to-your-ami) to specific AWS accounts.
+This is useful for testing, and for sharing private AMIs with specific accounts.
+
+If you want to make your AMIs public to the world, there's a shortcut:
+
+```
+cargo make ami-public
+```
+
+(Later, if you need to make the AMIs private again, you can do this.
+ The AMIs will then only be accessible to account IDs you've specifically granted.)
+
+```shell
+cargo make ami-private
+```
+
+## Promoting SSM parameters
+
+[Above](#making-your-amis-discoverable-with-ssm-parameters), we set SSM parameters based on our AMIs.
+The SSM parameter names include version numbers, which is handy for testing, but makes them hard to find by anyone else.
+Once we're satisfied, we can promote the SSM parameters to simpler names.
+
+```shell
+cargo make promote-ssm -e SSM_TARGET=latest
+```
+
+This will copy the fully versioned parameter from earlier, something like:
+
+```
+/your/prefix/here/aws-k8s-1.17/x86_64/1.0.1-dafe3b16/image_id
+```
+
+...to a simpler parameter name:
+```
+/your/prefix/here/aws-k8s-1.17/x86_64/latest/image_id
+```
+
+You can then use (and share) this parameter name to get the latest AMI ID.
```

Comment:
Outside of the `/aws` prefix, I don't believe parameters can be shared outside the account that owns the parameter.

Reply:
Yeah, but it's still useful to share within the account, e.g. to separate launch mechanisms from output of the deployment process.